data/json/monsters/cyborgs.json

```diff
@@ -238,5 +239,54 @@
       "PUSH_MON",
       "FILTHY"
     ]
+  },
+  {
+    "id": "mon_zomborg_devourer",
+    "type": "MONSTER",
+    "name": { "str": "zomborg devourer" },
+    "description": "Human bodies, robots, and cybernetic enhancements fused together into a colossus, with heads and limbs both biological and robotic sticking out of its bloated body at grotesque angles.  The nature of the thing makes it difficult to estimate how close it is to dying, and its capabilities might change if it manages to assimilate more zombies and cybernetics into itself.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "diff": 8,
+    "volume": "120 L",
+    "weight": "200 kg",
+    "species": [ "CYBORG", "ZOMBIE" ],
+    "hp": 112,
+    "speed": 60,
+    "material": [ "steel", "flesh" ],
+    "symbol": "Z",
+    "color": "light_gray",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 2,
+    "melee_dice_sides": 6,
+    "melee_damage": [ { "damage_type": "cut", "amount": 12 } ],
+    "armor_bash": 12,
+    "armor_cut": 28,
+    "armor_bullet": 8,
+    "bleed_rate": 25,
+    "vision_night": 3,
+    "weakpoint_sets": [ "wps_cyborg_body" ],
+    "families": [ "prof_wp_cyborg" ],
+    "harvest": "mon_zomborg",
+    "dissect": "dissect_mon_zomborg",
+    "special_attacks": [ [ "GRAB", 7 ], [ "scratch", 20 ], [ "ZOMBIE_FUSE", 80 ], [ "OPERATE", 30 ], [ "PARROT", 80 ] ],
```

Comment:
How will OPERATE function on a monster that isn't also PACIFIST like the defective nursebot? Isn't this just a delayed instakill?

Reply:
Yeah, in my testing it just complains about not having any anaesthetic. It's a fun idea but it seems like this isn't the right method

Reply:
Removing OPERATE